Description: Remove hashbang from bash completion file
 Remove the useless first line starting with the #! sequence from the
 scripts that is intended to be sourced.
Author: Gennaro Oliva <oliva.g@na.icar.cnr.it>
Forwarded: https://bugs.schedmd.com/show_bug.cgi?id=17036
Last-Update: 2021-05-17

--- a/contribs/slurm_completion_help/slurm_completion.sh
+++ b/contribs/slurm_completion_help/slurm_completion.sh
@@ -1,4 +1,3 @@
-#!/usr/bin/env bash
 ################################################################################
 # slurm_completion.sh - bash completion script for Slurm client commands.
 #
